pricelist_cache/tests/common.py

```diff
@@ -115,12 +110,12 @@ def setUpClass(cls):
             ]
         )
         # products
+        # TODO: create products instead of using odoo demo data
```

Comment:
for this, I would probably create a classmethod `setupExpectedValues`
which would do something like
```python3
cls.list_prices_mapping = {
    "pricelist_cache.list0": [
        {"id": cls.p6.id, "price": 100.0},
        {"id": cls.p7.id, "price": 15.8},
        {"id": cls.p8.id, "price": 100.0},
        {"id": cls.p9.id, "price": 47.0},
    ],
    ...
}
```
Then, in demo data, you can create those products (can't do that from tests, as pricelist demo are  referencing them).